lib/oxidized/input/ftp.rb

```diff
@@ -1,25 +1,21 @@
 module Oxidized
   require 'net/ftp'
   require 'timeout'
-  require_relative 'cli'
-
+  require 'oxidized/input/cli'
   class FTP < Input
     RescueFail = {
-      :debug => [
```

Comment:
They're doing nothing. they are commented out.

By all means leave them in. Personally didn't see much point them being there, if they are needed the user would still have to edit the code.